File: sdkwork-app-sdk-python/sdkwork_app_sdk/models/order_create_form.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class OrderCreateForm:
    """创建订单请求"""
    order_type: str
    product_id: str = None
    quantity: int = None
    items: List[OrderItemForm] = None
    address_id: str = None
    payment_method: str = None
    coupon_id: str = None
    remark: str = None
    source_channel: str = None
    recharge_points: int = None
    order_payload_valid: bool = None

File: sdkwork-app-sdk-kotlin/src/main/kotlin/com/sdkwork/app/OrderCreateForm.kt
package com.sdkwork.app

data class OrderCreateForm(
    val orderType: String? = null,
    val productId: String? = null,
    val quantity: Int? = null,
    val items: List<OrderItemForm>? = null,
    val addressId: String? = null,
    val paymentMethod: String? = null,
    val couponId: String? = null,
    val remark: String? = null,
    val sourceChannel: String? = null,
    val rechargePoints: Int? = null
)
